src/userAttrCache.h

```diff
@@ -175,7 +175,7 @@ class UserAttrCache: public std::map<UserAttrPair, std::shared_ptr<UserAttrCache
      * every time the attribute changes on the server and the new value is fetched.
      */
     Handle getAttr(uint64_t user, unsigned attrType, void* userp,
-                             UserAttrReqCbFunc cb, bool oneShot=false, uint64_t ph = Id::inval());
+                             UserAttrReqCbFunc cb, bool oneShot=false, bool fetch = true, uint64_t ph = Id::inval());
```

Comment:
I miss the documentation of the new param `fetch`

Reply:
Fixed